pkgs/top-level/python-packages.nix

```diff
@@ -24675,6 +24675,30 @@ in {
     doCheck = false;
   };
 
+  hieroglyph = buildPythonPackage (rec {
```

Comment:
please move this expression to `pkgs/development/python-modules/hieroglyph/default.nix`